CMakeLists.txt

```diff
@@ -206,6 +206,16 @@ endif()
 
 include(IrodsCPackCommon)
 
+set(CPACK_RPM_EXCLUDE_FROM_AUTO_FILELIST_ADDITION "/usr/sbin")
+# TODO: don't package the tests
```

Comment:
These comments are not really helpful after all. This could probably be squashed anyway

Reply:
Removed comment. May squash away later